providers/prometheus/go.mod

```
@@ -0,0 +1,9 @@
+module github.com/grpc-ecosystem/go-grpc-middleware/providers/prometheus
```

Since we want to release all modules once, we need v2 here.
```suggestion
module github.com/grpc-ecosystem/go-grpc-middleware/providers/openmetrics/v2
```

---

Okay, changed!